apps/fluent-tester/metro.config.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Metro configuration for React Native
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+const fs = require('fs');
+const path = require('path');
+const blacklist = require('metro-config/src/defaults/blacklist');
+const { getPackageInfo } = require('just-repo-utils');
+
+const rnPath = fs.realpathSync(path.resolve(require.resolve('react-native/package.json'), '..'));
+const rnwPath = fs.realpathSync(path.resolve(require.resolve('react-native-windows/package.json'), '..'));
+
+module.exports = {
+  watchFolders: [
+    ...getPackageInfo().dependencies().paths()
+  ],
+  resolver: {
+    extraNodeModules: {
+      // Redirect react-native to react-native-windows
+      'react-native': rnwPath,
+      'react-native-windows': rnwPath
+    },
+    // Include the macos platform in addition to the defaults because the fork includes macos, but doesn't declare it
+    platforms: ['ios', 'android', 'windesktop', 'windows', 'win32', 'web', 'macos'],
+    providesModuleNodeModules: ['react-native-windows'],
```

Comment:
will this work for win32? Currently it is set to @office-iss/react-native-win32

Reply:
I made a change so that if a bundle includes a platform it will try to load `metro.config.${platform}.js` before `metro.config.js`.  As a result in fluent-tester it will reroute to metro.config.win32.js